cmd/hvclient/flags.go

```diff
@@ -126,6 +126,9 @@ var (
 	fClaimRetrieve = flag.String("claimretrieve", "", "retrieve the domain claim with the specified ID")
 	fClaimSubmit   = flag.String("claimsubmit", "", "submit a domain claim for the specified domain")
 	fClaimDelete   = flag.String("claimdelete", "", "delete the domain claim with the specified ID")
-	fClaimDNS      = flag.String("claimdns", "", "request assertion of domain control for the domain claim with the specified ID")
+	fClaimDNS      = flag.String("claimdns", "", "request assertion of domain control using DNS for the domain claim with the specified ID")
+	fClaimHTTP     = flag.String("claimhttp", "", "request assertion of domain control using HTTP for the domain claim with the specified ID")
+	fScheme        = flag.String("scheme", "http", "protocol used to verify assertion of domain control using HTTP for the domain claim")
```

Comment:
Should the default flag value be https instead of http? Assuming this flag is most of the time for specifying either http or https, I think it'd be good to encourage it as the default.